package.json

```diff
@@ -78,6 +78,7 @@
     "express": "^4.12.3",
     "filesize": "^3.1.3",
     "find-up": "^2.1.0",
+    "find-yarn-workspace-root": "1.0.0",
```

Comment:
nitpick: looking at [`find-yarn-workspace-root`'s `package.json`](https://github.com/square/find-yarn-workspace-root/blob/master/package.json). do you mind adding [`files` option](https://docs.npmjs.com/files/package.json#files) to it? we did have issues with `ember-cli` download size before, just didn't want us to slowly fall in the same pitfall.

Reply:
Changed here: https://github.com/square/find-yarn-workspace-root/pull/5

Reply:
@thetimothyp - Sorry for the delayed review here, would you mind updating to `^1.0.0` in the `package.json`? That would allow us to get updates out if needed...